src/Faithlife.WebRequests/Json/AutoWebServiceResponse.cs

```diff
@@ -10,6 +10,21 @@ namespace Faithlife.WebRequests.Json
 	/// </summary>
 	public abstract class AutoWebServiceResponse
 	{
+		/// <summary>
+		/// Gets the HTTP headers on the response.
+		/// </summary>
+		public HttpResponseHeaders Headers { get; private set; }
+
+		/// <summary>
+		/// Gets whether or not the response's content was able to be mapped to a strongly typed property corresponding to the HTTP status code.
+		/// </summary>
+		public bool IsStatusCodeHandled { get; private set; }
```

Comment:
I don't understand the use-case for this property.

I, as a consumer, don't care as much about "Does the `StatusCode` match one of the properties on the `AutoWebServiceResponse` class" as I care about "Does the `StatusCode` match one of the properties that *my code knows how to handle*" -- which is a subset of the former. A new version of the client can even change the fromer out from under me, while it cannot change the latter, and so I wouldn't feel comfortable trusting this property.

If I *really* wanted to get at this boolean, even without this property I still could -- by checking against all the properties I know about.